packages/features/bookings/lib/handleCancelBooking.ts

```diff
@@ -120,8 +119,12 @@ async function handler(req: NextApiRequest & { userId?: number }) {
     },
   });
 
-  const attendeesListPromises = bookingToDelete.attendees.map(async (attendee) => {
-    return {
+  const teamMembersPromises = [];
+  const attendeesListPromises = [];
+  const hostsPresent = !!bookingToDelete.eventType?.hosts;
+
+  for (const attendee of bookingToDelete.attendees) {
+    const attendeeObject = {
```

Comment:
Since all team members are stored as attendees in the booking, we can create the attendee object now.